src/dlgTriggerEditor.cpp

```diff
@@ -531,8 +531,8 @@ dlgTriggerEditor::dlgTriggerEditor(Host* pH)
     // QLineEdit does not provide a signal to hook on for the clear action
     // see https://bugreports.qt.io/browse/QTBUG-36257 for problem
     // credit to Albert for the workaround
-    for (int i(0); i < pLineEdit_searchTerm->children().size(); ++i) {
-        auto *pAction_clear(qobject_cast<QAction *>(pLineEdit_searchTerm->children().at(i)));
+    for (auto i : pLineEdit_searchTerm->children()) {
```

Comment:
Might I suggest `child` rather than `i` as the iteration value - it is no longer an integer...?

Reply:
Great suggestion